### PR TITLE
Partial backport of #12564

### PR DESCRIPTION
**Pull request by olemarkus**
Partial backport of #12564 
Keep CNP status updates enabled to avoid BC break.

### CODE REVIEW

File: pkg/model/components/cilium.go [hunk 1]
```diff
@@ -139,6 +139,10 @@ func (b *CiliumOptionsBuilder) BuildOptions(o interface{}) error {
 		c.EnableL7Proxy = fi.Bool(true)
 	}
 
+	if c.DisableCNPStatusUpdates == nil {
+		c.DisableCNPStatusUpdates = fi.Bool(false)
+	}
+
```
Comment (hakman) [inline pkg/model/components/cilium.go:145]: This should be the only change to this file, right?

**Review by hakman — CHANGES_REQUESTED** [review]

File: pkg/model/components/cilium.go (hunk 1 above)
Comment (olemarkus) [reply pkg/model/components/cilium.go:145]: VS Code started autoformatting some new stuff on save. I just let it. Seems to be improvements (I am really inconsistent when it comes to whitespace usage, so I like this).

Comment (hakman) [reply pkg/model/components/cilium.go:145]: Was just thinking that this would make cherry-picks harder later.

Comment (olemarkus) [reply pkg/model/components/cilium.go:145]: Probably not. But anyways, removed those changes now

**Review by hakman — APPROVED** [review]

**Comment by yurrriq** [comment]
Will this land in 1.22.2 then? :crossed_fingers:

**Comment by hakman** [comment]
> Will this land in 1.22.2 then? 🤞

yes

**Pull request merged**